kube/Cargo.toml

```diff
@@ -29,6 +29,7 @@ admission = ["kube-core/admission"]
 derive = ["kube-derive"]
 config = ["kube-client/config"]
 runtime = ["kube-runtime"]
+cache = ["kube-runtime/controller"]
```

Comment:
if we have separate controller and reflector features, we should probably have the same features in kube to avoid people having to fallback to importing multiple crates separately